### ISSUE TITLE
Support SerenityOS

### ISSUE THREAD

**Issue by XVilka** [rizinorg/ideas#39]

**Comment by historicattle** [comment]
I have added the rizin port to serenity! [serenity#26589](https://github.com/SerenityOS/serenity/pull/26589). Should i create a PR to merge changes in rizin? @notxvilka

**Comment by notxvilka** [comment]
> I have added the rizin port to serenity! [serenity#26589](https://github.com/SerenityOS/serenity/pull/26589). Should i create a PR to merge changes in rizin? [@notxvilka](https://github.com/notxvilka)

Yes. Please do. But better to do that once https://github.com/rizinorg/rizin/pull/5880 is merged, since I see there is a minor change in the heap parser.

**Comment by historicattle** [comment]
Okay

**Comment by notxvilka** [comment]
@historicattle I merged that glibc heap PR, so it's a good time for sending a PR.
